vuln/npm/457.json

```diff
@@ -0,0 +1,26 @@
+{
+  "id": 457,
+  "title": "Last Will message is not checked against authorization",
+  "overview": "Aedes does respect its own authorization rules when a client set a Last Will",
+  "created_at": "2018-08-07",
+  "updated_at": "2018-08-07",
+  "publish_date": "2018-08-07",
+  "author": {
+    "name": "Matteo Collina",
+    "website": null,
+    "username": "mcollina"
+  },
+  "module_name": "aedes",
+  "cves": [
+  ],
+  "vulnerable_versions": "<0.35.0",
+  "patched_versions": ">=0.35.1",
+  "recommendation": "Update aedes module to version >= 0.35.1",
+  "references": [
+    "https://github.com/mcollina/aedes/issues/211",
+    "https://github.com/mcollina/aedes/issues/212"
+  ],
+  "cvss_vector": "CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:C/C:N/I:L/A:N/E:H/RL:O/RC:C/CR:X/IR:X/AR:X/MAV:X/MAC:L/MPR:L/MUI:N/MS:C/MC:N/MI:L/MA:N",
```

Comment:
we usually just score them with the base score and without environmental or other affects, so this should probably read: `CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:C/C:N/I:L/A:N` and that scores 5.0